secure_message/resources/messages.py

```diff
@@ -35,16 +35,15 @@ def get():
         message_args = get_options(request.args)
 
         message_service = Retriever()
-        status, result = message_service.retrieve_message_list(message_args.page, message_args.limit, g.user,
-                                                               ru_id=message_args.ru_id, survey=message_args.survey,
-                                                               cc=message_args.cc, label=message_args.label,
-                                                               descend=message_args.desc, ce=message_args.ce)
-
-        if status:
-            resp = paginated_list_to_json(result, message_args.page, message_args.limit, request.host_url,
-                                          g.user, message_args.string_query_args, MESSAGE_LIST_ENDPOINT)
-            resp.status_code = 200
-            return resp
+        result = message_service.retrieve_message_list(message_args.page, message_args.limit, g.user,
+                                                       ru_id=message_args.ru_id, survey=message_args.survey,
```

Comment:
In the rest of the services this is referred to as ru_ref or business_id.

Reply:
It needs a wider discussion as to ru-id/ru ref and business ref which is outside of the scope of this pr and one for the tech leads